Control coverage/Feature/CosmosDB.md

```diff
@@ -269,3 +270,56 @@ Threat Protection for Azure Cosmos DB provides an additional layer of security i
 
 ___ 
 
+## Azure_CosmosDB_DP_Use_Secure_TLS_Version
+
+### Display Name 
+Use approved version of TLS for the Cosmos DB
+
+### Rationale 
+TLS provides confidentiality and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions.
+
+### Control Settings 
+```json 
+{
+    "MinReqTLSVersion": "1.2"
+}
+ ```  
+### Control Spec 
+
+> **Passed:** 
+> Minimum TLS version is set to 1.2 or higher for Cosmos DB.
+> 
+> **Failed:** 
+> Any of the following conditions is met.
+> * Minimum TLS version is not set (default 1.0).
+> * Minimum TLS version is less than 1.2 (configured min required TLS version) for Cosmos DB.
```

Comment:
same as above. let's not mention 1.2 but req only

Reply:
done